smithy-diff/src/test/java/software/amazon/smithy/diff/evaluators/RemovedOperationErrorTest.java

```diff
@@ -52,6 +52,8 @@ public void detectsRemovedErrors() {
 
         // Emits an event for each removal.
         assertThat(TestHelper.findEvents(events, "RemovedOperationError").size(), equalTo(2));
+        assertThat(TestHelper.findEvents(events, "RemovedOperationError.foo.baz#E1").size(), equalTo(1));
```

Comment:
Similar suggestion from the other PR - errors have to have unique names across the service closure. We can just use the shape ID shape name here (e.g., `RemovedOperationError.E1` / `AddedOperationError.E1`).